pydantic/typing.py

```diff
@@ -193,3 +193,12 @@ def update_field_forward_refs(field: 'Field', globalns: Any, localns: Any) -> No
     if field.sub_fields:
         for sub_f in field.sub_fields:
             update_field_forward_refs(sub_f, globalns=globalns, localns=localns)
+
+
+def get_class(type_: AnyType) -> Optional[Union[AnyType, bool]]:
```

Comment:
```suggestion
def get_class(type_: AnyType) -> Union[None, bool, AnyType]]:
```
Otherwise I think mypy will complain.